src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -212,6 +212,8 @@ public ReadOnlyBooleanProperty gameRunningProperty() {
       return completedFuture(null);
     }
 
+    newGameInfo.setMap("NeroxisMapGenerator_1_0.0.1-prealpha");
```

Comment:
This seems t be debug code please delete it

Reply:
euhm srsly, there seem to be some leftovers in code I didn't touch for the past 4 months, thx for paying that much attention :)